donor-portal/src/pages/newsroom/index.tsx

```diff
@@ -5,6 +5,7 @@ import pressReleaseEnglish from "./static/press-release-english.pdf";
 import pressReleaseSinhala from "./static/press-release-sinhala.pdf";
 import pressReleaseTamil from "./static/press-release-tamil.pdf";
 import "./styles.css";
+import kandy_donation_image from "./images/kandy_donation.jpg";
```

Comment:
```suggestion
import kandyDonationImage from "./images/kandy_donation.jpg";
```
Let's use camel case.